organization.go

```diff
@@ -1,25 +1,117 @@
 package zd
 
+import (
+	"fmt"
+)
+
+type OrganizationWrapper struct {
+	Organization *Organization `json:"organization"`
+}
+
 type OrganizationResponse struct {
```

Comment:
That's the same struct duplicated. Maybe we should get rid of the OrganizationResponse and use the wrapper instead.